src/main/java/nl/novi/behuiswerkles11/mappers/TelevisionModelMapper.java

```diff
@@ -0,0 +1,70 @@
+package nl.novi.behuiswerkles11.mappers;
+
+import nl.novi.behuiswerkles11.dtos.TelevisionDto;
+import nl.novi.behuiswerkles11.dtos.TelevisionRequestDto;
+import nl.novi.behuiswerkles11.models.Television;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class TelevisionModelMapper {
+
+    //    Mapper functie om de tv om te zetten naar een dto tv, die we willen teruggeven aan de gebruiker
+    public TelevisionDto translateToDto(Television television) {
+        TelevisionDto dto = new TelevisionDto();
+
+        dto.setName(television.getName());
+        dto.setBrand(television.getBrand());
+        dto.setPrice(television.getPrice());
+        dto.setAvailableSize(television.getAvailableSize());
+        dto.setRefreshRate(television.getRefreshRate());
+        dto.setScreenType(television.getScreenType());
+        dto.setScreenQuality(television.getScreenQuality());
+        dto.setSmartTv(television.getSmartTv());
+        dto.setWifi(television.getWifi());
+        dto.setVoiceControl(television.getVoiceControl());
+        dto.setHdr(television.getHdr());
+        dto.setBluetooth(television.getBluetooth());
+        dto.setAmbiLight(television.getAmbiLight());
+
+        return dto;
+    }
+
+    public List<TelevisionDto> translateToDtos(List<Television> televisions) {
+        List<TelevisionDto> result = new ArrayList<>();
+        for (Television television: televisions) {
+            result.add(translateToDto(television));
+        }
+        return result;
+    }
+
+    //    Mapper functie om de input/request om te zetten naar een tv
+    public Television createTelevision(TelevisionRequestDto dto) {
```

Comment:
Ik zou deze ook "translateToTelevision" noemen, zodat de naamsconventie in 1 lijn is met je andere methodes. Hoe je het noemt maakt niet veel uit, belangrijk is wel dat je uniform bent